[2022-09-05T18:11:06.901+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: our_dag_with_python_operator_06.get_age manual__2022-09-05T18:11:06.077996+00:00 [queued]>
[2022-09-05T18:11:06.921+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: our_dag_with_python_operator_06.get_age manual__2022-09-05T18:11:06.077996+00:00 [queued]>
[2022-09-05T18:11:06.922+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-05T18:11:06.922+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 6
[2022-09-05T18:11:06.922+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-05T18:11:06.944+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): get_age> on 2022-09-05 18:11:06.077996+00:00
[2022-09-05T18:11:06.949+0000] {standard_task_runner.py:52} INFO - Started process 3921 to run task
[2022-09-05T18:11:06.953+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_dag_with_python_operator_06', 'get_age', 'manual__2022-09-05T18:11:06.077996+00:00', '--job-id', '335', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python_operator.py', '--cfg-path', '/tmp/tmphsm28bmz', '--error-file', '/tmp/tmp9_3e93_a']
[2022-09-05T18:11:06.955+0000] {standard_task_runner.py:80} INFO - Job 335: Subtask get_age
[2022-09-05T18:11:07.045+0000] {task_command.py:371} INFO - Running <TaskInstance: our_dag_with_python_operator_06.get_age manual__2022-09-05T18:11:06.077996+00:00 [running]> on host c033faf33c57
[2022-09-05T18:11:07.157+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ancordss
AIRFLOW_CTX_DAG_ID=our_dag_with_python_operator_06
AIRFLOW_CTX_TASK_ID=get_age
AIRFLOW_CTX_EXECUTION_DATE=2022-09-05T18:11:06.077996+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-05T18:11:06.077996+00:00
[2022-09-05T18:11:07.181+0000] {python.py:173} INFO - Done. Returned value was: None
[2022-09-05T18:11:07.202+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=our_dag_with_python_operator_06, task_id=get_age, execution_date=20220905T181106, start_date=20220905T181106, end_date=20220905T181107
[2022-09-05T18:11:07.251+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-05T18:11:07.306+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
